lib/gui/etcher.ts

```diff
@@ -43,7 +43,7 @@ async function checkForUpdates(interval: number) {
 				const release = await autoUpdater.checkForUpdates();
 				const isOutdated =
 					semver.compare(release.updateInfo.version, version) > 0;
-				const shouldUpdate = release.updateInfo.stagingPercentage || 0 > 0;
```

Comment:
`...  || 0 > 0`

Yeah, you can see why that didn't work, if it was expecting zero to be bigger than zero :rofl: